GWFish/modules/waveforms.py

```diff
@@ -286,8 +286,8 @@ def _setup_lal_caller_args(self):
             self._waveform_postprocessing = self._hf_postproccessing_SimInspiralFD
             self._lalsim_caller = lalsim.SimInspiralFD
             self._lalsim_args = [
-                self.gw_params['mass_1'] * lal.MSUN_SI * (1 + self.gw_params['redshift']),  # in [kg]
-                self.gw_params['mass_2'] * lal.MSUN_SI * (1 + self.gw_params['redshift']),  # in [kg]
+                self.gw_params['mass_1'] * lal.MSUN_SI,# * (1 + self.gw_params['redshift']),  # in [kg]
```

Comment:
I think we can fully remove it, not just comment it (less confusion for people who come later)

Reply:
Done!